### DISCUSSION TITLE
Log link in emails link not consistent with url

### DISCUSSION THREAD

**Discussion by saswatm11** [apache/airflow#35840] [Q&A]
Hello

Running a LocalExecutor airflow instance on a Linux host. When a job fails an email is sent but most of the times the log link is set as "http://localhost:port" as opposed to the value set in base_url in the airflow.cfg file.

How can I ensure that the log links are always as "http://hostname:port"?

Thanks

**Comment by potiuk** [comment]
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#hostname-callable
